copy-common.mk

```diff
@@ -48,6 +57,23 @@ endif
 
 post-clean::
 
+# External targets that can be used, but should not be overridden
+
+_checkcommonupdate::
+	@TMP_FILE=`mktemp copy-common.mk.XXXXXX`; \
+		curl -o $${TMP_FILE} -s -S -L https://raw.github.com/AchievementNetwork/common-make/main/copy-common.mk; \
+		if [ $$? -ne 0 ]; then echo "Update check failed"; exit 1; fi; \
+		diff -b --brief copy-common.mk $${TMP_FILE} > /dev/null; \
+		if [ $$? -ne 0 ]; then echo "It looks like copy-common.mk is out of date.  Please run \`make _commonupdate\`"; else echo "copy-common.mk up to date"; fi; \
```

Comment:
Is it possible/desirable to exit with an error code if it's out of date, for scripting purposes?

Reply:
We'd have to decide different error codes for "I couldn't do the check" vs. "The check showed out of date".  I'd like to see how this turns out in practice a bit before we do this.  I mostly see this as something that is done interactively at the moment rather than scripted.  If we did want to script it, then maybe the script just uses the _commonupdate target to always attempt the update and then only commit if git shows changes (i.e., this target would not be used in such a case).